docs/guides/connecting_to_your_data/database/components/_datasource_redshift_test.md

```diff
@@ -0,0 +1,32 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import TechnicalTag from '@site/docs/term_tags/_tag.mdx';
+
+Verify your new <TechnicalTag tag="datasource" text="Datasource" /> by loading data from it into a <TechnicalTag tag="validator" text="Validator" /> using a `BatchRequest`.
+
+<Tabs
+  defaultValue='runtime_batch_request'
+  values={[
+  {label: 'Using a SQL query', value:'runtime_batch_request'},
+  {label: 'Using a table name', value:'batch_request'},
+  ]}>
+  
+<TabItem value="runtime_batch_request">
+
+Here is an example of loading data by specifying a SQL query.
+
+```python file=../../../../../tests/integration/docusaurus/connecting_to_your_data/database/redshift_yaml_example.py#L56-L70
```

Comment:
Things are really clear and since `redshift` has been under docs-integration tests for a while now, I don't really have concerns on the accuracy/up-to-dateness of this doc. 

However, it would really be nice to get these under named snippets. Would it be possible to do the work as part of the PR?

Reply:
Since the work will end up touching multiple documents, I would prefer to put it in its own PR, and we do have plans to do a snippet overhaul for all the existing docs that use line number snippets.